app/views/posts/_schema.html.erb

```diff
@@ -0,0 +1,27 @@
+<% if post && post.post_type.name == 'Question' %>
+  <% suggested_answers = @children.filter { |answer| answer.score > 0.5 } %>
+  <script type="application/ld+json">
+    {
+      "@context": "https://schema.org",
+        "@type": "QAPage",
+        "mainEntity": {
+        "@type": "Question",
+        "name": "<%= @post.title %>",
+        "text": "<%= @post.body.gsub(/\r?\n/, '') %>",
+        "answerCount": <%= @children&.length || 0 %>,
+        "upvoteCount": <%= @post.upvote_count %>,
+        "suggestedAnswer": [
+        <% suggested_answers.each_with_index do |answer, i| %>
+          <% if i > 0 then %>,<% end %>
```

Comment:
`<% suggested_answers.first %>` should work here?

Reply:
Not quite sure how this would work? This is just to get a comma-separated array for the JSON, since the Google analyzer complained when I had a trailing comma.

Reply:
Unfortunately, that doesn't work, and it looks like rails doesn't have a cleaner way to do it other than manually checking